CaPPMS/Data/FaqManagerService.cs

```diff
@@ -0,0 +1,152 @@
+﻿using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.IO;
+using System.Security.Principal;
+using System.Threading.Tasks;
```

Comment:
Quite a few of these directives just are needed and do not belong here. Please sort this out.